docs/mesos/MesosClusterPreparation.md

```diff
@@ -0,0 +1,139 @@
+
+    rpm -Uvh http://repos.mesosphere.com/el/7/noarch/RPMS/mesosphere-el-repo-7-1.noarch.rpm
+
+## ZooKeeper
+Install Mesos' datastore - ZooKeeper:
+
+```
+yum install -y mesosphere-zookeeper
+systemctl start zookeeper
+```
+
+ZooKeeper uses tcp over port 2181. If you're using a firewall, open this port:
+
+```
+sudo firewall-cmd --zone=public --add-port=2181/tcp --permanent
+sudo systemctl restart firewalld
+```
+
+## Mesos Master
+Supported Versions: `0.28` (recommended, required for Docker Containerizer),
+`v0.27`, `v0.26.0` (deprecated)
+
+With Zookeeper up and running, we can launch our mesos-master process. Run the following commands, replacing `$IP` with the cluster-accessible IP you would like Master to bind to:
+
+```
+yum install -y mesos
+echo $IP > /etc/mesos-master/hostname
+echo $IP > /etc/mesos-master/ip
+systemctl start mesos-master
+```
+
+## Marathon
+Supported Versions: `v1.0.0` (recommended), `v0.14.0` - `v0.14.2`
+
+Once mesos-master is running, we can launch Marathon:
+
+```
+sudo yum install -y marathon
+systemctl start marathon
+```
+
+Marathon listens for tcp connections on port 8080. If you're using a firewall,
+open this port:
+
+```
+sudo firewall-cmd --zone=public --add-port=8080/tcp --permanent
+sudo systemctl restart firewalld
+```
+
+## etcd
+Calico uses etcd as its data store and communication mechanism among Calico components. Install it from Centos' repositories with the following command, replacing `$IP` with the cluster-accessible IP you would like etcd to bind to:
+
+```
+yum install -y etcd
+echo ETCD_LISTEN_CLIENT_URLS=\"http://0.0.0.0:2379\" >> /etc/etcd/etcd.conf
+echo ETCD_ADVERTISE_CLIENT_URLS=\"http://$IP:2379\" >> /etc/etcd/etcd.conf
+systemctl start etcd.service
+```
+
+Etcd listens for tcp connections on port 2379. If you're using a firewall,
+open this port:
+
+```
+sudo firewall-cmd --zone=public --add-port=2379/tcp --permanent
+sudo systemctl restart firewalld
+```
+
+## Mesos Agents
+The Mesos Agent dependencies differ depending on the type of
+containerizer in use.
+
+#### Unified Containerizer
+You will need to install `net-modules` on your agents for the
+Unified Containerizer to work. Check out the
+[net-modules install guide](./net-modules/README.md).
+
+#### Docker Containerizer
+The Docker containerizer requires Docker 1.10+ to be installed:
+
+```
+yum install docker-engine
+systemctl docker start
+```
+
+#### All Agent Types
+After configuring the containizer requirements,
+install the base`mesos-slave` service:
+
+```
+yum install -y mesos
+sh -c 'echo zk://#{master_ip}:2181/mesos > /etc/mesos/zk'
+sh -c 'echo #{ip} > /etc/mesos-slave/ip'
+sh -c 'echo #{ip} > /etc/mesos-slave/hostname'
+sh -c 'echo mesos,docker > /etc/mesos-slave/containerizers'
+systemctl start mesos-slave.service"
+```
+
+## Next steps
+Install Calico by following our [Manual Install Guide](ManualInstallCalicoMesos.md).
```

Comment:
Doesn't really make sense to point them at the manual install netmodules guide from here.